web/src/components/MailBox/MailItem.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
+import Form from 'react-bootstrap/Form'
+import ListGroup from 'react-bootstrap/ListGroup'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+
+const MailItem = ({ id, subject, receivedOn, from, read, selected, onSelect, onClick }) => {
+  dayjs.extend(relativeTime)
```

Comment:
Wouldn't it be simpler to move the initialization above the component so it doesn't re initialize each time the component is rendered?

Reply:
You're right. It's done. I let you resolve and merge.